js/src/components/tree-select-control/tags.js

```diff
@@ -37,7 +37,7 @@ const Tags = ( { tags, disabled, onChange = () => {} } ) => {
 	return (
 		<div className="woocommerce-tree-select-control__tags">
 			{ tags.map( ( item, i ) => {
-				if ( ! item.label || ! item.id ) {
+				if ( ! item.label ) {
```

Comment:
❔ 
I don't quite catch what the original logic was intended to deal with, so this comment may be pointless. But if the logic was removed only to allow the option with `value: ''`, it could also consider replacing`! item.id` with `! item.hasOwnProperty( 'id' )`.

Reply:
This is to show the label with value as '' yes. 

The original logic was because I never thought we wanted to support '' as a value.

Reply:
Got it, thanks!